src/constituency_labeler/TurboConstituencyLabeler.cpp

```diff
@@ -47,6 +47,9 @@ int main(int argc, char** argv) {
     TestConstituencyLabeler();
   }
 
+  // Destroy allocated memory regarding line flags.
+  google::ShutDownCommandLineFlags();
+  google::ShutdownGoogleLogging();
```

Comment:
I only added them to reduce gflag "errors" during valgrind memory leak analysis.